src/Orchard.Environment.Shell/ShellOptions.cs

```diff
@@ -1,7 +0,0 @@
-﻿namespace Orchard.Environment.Shell
-{
-    public class ShellOptions
```

Comment:
Not sure it should have changed. This is becoming the options for when the shells are loaded from the file system in "Sites", it's an option for a specific implementation.